configuration.example.yaml

```diff
@@ -5,6 +5,9 @@ server:
   address: 127.0.0.1
   # Port on which the webserver listens
   port: 7200
```

Comment:
Should we clarify here that the `address` and `port` settings are not used when the systemdSocketActivation is configured? Or add that as a further sentence to `systemdSocketActivation`?